apps/wix/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@contentful/wix-integration",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@contentful/app-sdk": "4.22.0",
+    "@contentful/f36-components": "4.45.0",
+    "@contentful/f36-tokens": "4.0.2",
+    "@contentful/react-apps-toolkit": "1.2.16",
+    "contentful-management": "10.38.3",
+    "emotion": "10.0.27",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-router": "6.14.1",
+    "react-router-dom": "6.14.1"
+  },
+  "scripts": {
+    "install-ci": "npm ci",
+    "start": "vite",
+    "lint": "prettier --write .",
+    "build": "tsc && vite build",
+    "test": "vitest run",
+    "test:watch": "vitest",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "upload": "contentful-app-scripts upload --bundle-dir ./dist",
```

Comment:
Just an FYI, we will add in a deploy script with your app definition id for our Contentful Marketplace Apps Org in a subsequent PR.